clean_files.txt

```diff
@@ -24,12 +24,6 @@ hooks
 .gitattributes
 lint_clean_files.sh
 
-# plugins
```

Comment:
oops :smile:

Reply:
hah. I think this was my fault in the end though :upside_down_face: